### ISSUE TITLE
Gutenberg editor styles: ID prefix necessary?

### ISSUE THREAD

**Issue by luminuu** [MapSteps/Page-Builder-Framework#34]
I'm currently working on a project where I created a child theme for the WP Page Builder Framework theme. I'd like to overwrite a few styles in the editor styles file, but in the theme editor styles some of the selectors are prefixed with an ID:

https://github.com/MapSteps/Page-Builder-Framework/blob/11c7764b19b6bb7e9bf326076ce6c2f9357b50d6/css/block-editor-styles.css#L27-L44

When I use the `add_editor_style()` function with the `add_action('after_setup_theme')`, it automatically prefixes every element with `.editor-styles-wrapper`. Due to the higher specificity in the WPBF editor styles, it is impossible to overwrite those styles. I do not want to use `!important` in this case.

I did not find any difference when removing the `#wpwrap` IDs in the editor styles file, so I'm wondering if these are still necessary or if they can be removed.

**Comment by mapsteps** [comment]
Hi @luminuu,

That's most likely from one of the earlier version of Gutenberg we've integrated with. Overriding Gutenberg's default styles hasn't always been straightforward and often required us to come up with some workarounds like - in this case - adding the ID before targeting `.editor-styles-wrapper`.

I can't say if removing the ID won't cause other issues and I don't remember exactly why we had to add the ID but it is there for a reason.

Things might have changed with the recent Gutenberg updates but I'll definitely have to go through the entire integration we have to make sure everything runs fine after removing the ID.

When we came up with this I think there was no documentation on how to properly override Gutenberg core styles and I'll have to check if those exist today.

Definitely going to look into this!
